src/Components/GrowthChart/GrowthGraph.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { DashboardGraphChart } from './GrowthGraphChart';
+import zscore0to14Weeks from './GrowthStandardsData/HeadCircumferenceForAgeBoys0to13weeks.json'
+import percent24to60Months from './GrowthStandardsData/HeightForAgeBoys2to5years.json'
+import heightboy2to5 from './GrowthStandardsData/fakeboyheight2to5.json'
+import { useIdentifyPercentileOrZScore } from "./Hooks/useIdentifyPercentileOrZScore";
```

Comment:
We have an index file
```suggestion
import { useIdentifyPercentileOrZScore } from "./Hooks";
```